codegens/curl/lib/index.js

```diff
@@ -31,14 +32,17 @@ self = module.exports = {
     if (timeout > 0) {
       snippet += ` ${form('-m', format)} ${timeout}`;
     }
+    // eslint-disable-next-line no-useless-escape
+    if ((url.match(/[\{\[]/g) || []).length > 1) {
```

Comment:
Remove unnecessary escape characters. `(/[{[]/g)` <- this should work the same way.

Reply:
No actually they are not part of a string. Those escape characters are for the regex

Reply:
Those are redundant escape characters, not required for the regex. Removing them the regex will work the same way.

Reply:
`// eslint-disable-next-line no-useless-escape` 
Avoid disabling any lint rules as far as possible.

Reply:
> Those are redundant escape characters, not required for the regex. Removing them the regex will work the same way.

yeah they work without escape character too. Thanks